packages/common/src/core/schemas/types.ts

```diff
@@ -72,3 +72,10 @@ export type UiSchemaElementOptions = Pick<
   IUiSchemaElement,
   "scope" | "options"
 >;
+
+export interface IUiSchemaComboboxItem {
```

Comment:
Not part of this work, but we really need a chore to remove the duplicated configuration editor types/interfaces in this file from `opendata-ui` and to update all of the imports to use the ones here.

Reply:
Yes. My next PR to -ui will swap to importing them from hub.js